server/data/auth/pickem-597ad-firebase-adminsdk-ssuf8-d551e72305.js

```diff
@@ -1,12 +1,12 @@
 module.exports = {
-  type: process.env.REACT_APP_TYPE,
```

Comment:
Good call, I would assume prefixing each with `REACT_APP` is only necessary on the front end.